tests/unit/api/bucketPolicyAuth.js

```diff
@@ -35,6 +36,9 @@ const basePolicyObj = {
 };
 const bucketName = 'matchme';
 const log = new DummyRequestLogger();
+const publicUserAuthInfo = new AuthInfo({
+    canonicalID: 'http://acs.amazonaws.com/groups/global/AllUsers',
```

Comment:
Can you pick this from constants: https://github.com/scality/cloudserver/blob/77d41dffcf6d59a9623586bde51f5adf2ac8bcc1/constants.js#L47